components.json

```diff
@@ -48,12 +48,13 @@
 		},
 		"markup": {
 			"title": "Markup",
-			"alias": ["html", "xml", "svg", "mathml"],
+			"alias": ["html", "xml", "svg", "mathml", "vue"],
 			"aliasTitles": {
 				"html": "HTML",
 				"xml": "XML",
 				"svg": "SVG",
-				"mathml": "MathML"
+				"mathml": "MathML",
+				"vue": "Vue",
```

Comment:
The comma at the end of this line is invalid JSON. That's why the build is failing.

Reply:
Fixed.